api/model/src/main/java/org/projectnessie/model/ContentMetadata.java

```diff
@@ -15,27 +15,35 @@
  */
 package org.projectnessie.model;
 
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonInclude.Include;
 import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
 import com.fasterxml.jackson.databind.annotation.JsonSerialize;
+import javax.annotation.Nullable;
 import javax.validation.constraints.NotEmpty;
 import org.eclipse.microprofile.openapi.annotations.enums.SchemaType;
 import org.eclipse.microprofile.openapi.annotations.media.Schema;
 import org.immutables.value.Value;
 
 @Value.Immutable
-@JsonSerialize(as = ImmutableGenericMetadata.class)
-@JsonDeserialize(as = ImmutableGenericMetadata.class)
-public interface GenericMetadata {
+@JsonSerialize(as = ImmutableContentMetadata.class)
+@JsonDeserialize(as = ImmutableContentMetadata.class)
+public interface ContentMetadata {
```

Comment:
Why do we have custom server-side serializers for `Content`, but no for `ContentMetadata`?

Reply:
I mean if plain Json is ok for metadata, why not for `Content`?

Reply:
I'm actually thinking about whether it makes sense for `Content`. At least clients could then receive unknown content-types without crashing during deserialization.